core/cas-server-core-webflow-api/src/main/java/org/apereo/cas/web/flow/configurer/AbstractCasMultifactorWebflowConfigurer.java

```diff
@@ -31,6 +34,9 @@
  */
 @Slf4j
 public abstract class AbstractCasMultifactorWebflowConfigurer extends AbstractCasWebflowConfigurer {
+
+    private static final String MFA_INITIALIZE_BEAN_ID = "mfaInitializeAction";
```

Comment:
This likely need to move next to other SWF constants in form of a ACTION_ID_XYZ.

Reply:
Ok, should we move the same constants in AbstractMultifactorTrustedDeviceWebflowConfigurer to CasWebflowConstants as well?

Reply:
Yes please.